Readme.md

```diff
@@ -10,6 +10,15 @@
 2. `yarn && yarn start`
 3. <http://localhost:3333/> should open automatically.
 
+## Contributing 
+
+We welcome contributions to the docmentation site! Please test your changes locally followed by a PR against our `master` branch. After your PR is reviewed, it will be merged and the branch will be deleted. 
```

Comment:
What's our timeline on using `main`, instead?

Can we clarify use of the word "test"? Does it just mean to standup the docs site, and visually evaluate the rendered change, locally?

Reply:
Followed up about `main` internally. 
```suggestion
We welcome contributions to the documentation site! Please verify your changes locally followed by a PR against our `master` branch. After your PR is reviewed and all tests pass, it will be merged and the branch will be deleted. 
```